plugins/hazelcast39/src/main/java/org.radargun.service/Hazelcast39CounterOperations.java

```diff
@@ -0,0 +1,55 @@
+package org.radargun.service;
+
+import com.hazelcast.core.IAtomicLong;
+import org.radargun.traits.CounterOperations;
+
+/**
+ * @author Martin Gencur
+ */
+public class Hazelcast39CounterOperations implements CounterOperations {
```

Comment:
Hazelcast supports IAtomicLong since 3.3 (or maybe earlier), so you can include it for earlier versions as well.